src/account-info.js

```diff
@@ -23,6 +23,17 @@ function AccountInfo (object) {
   this._isMobileVerified = Boolean(object.sms_verified);
 
   this._currency = object.currency;
+
+  var notifications = {};
+  if (object.notifications_type) {
+    object.notifications_type.forEach(function (code) {
+      var type = Math.log2(code);
+      if (type === 0) notifications.email = true;
+      if (type === 2) notifications.http = true;
+      if (type === 5) notifications.sms = true;
```

Comment:
Instead of looping over the `notifications_type` object and then setting `.email`, `.http` and `.sms`, it's more readable like this:
```js
email = function (o) { .... ; ... === 0}
http = ...
sms = ...
{
  email: Boolean(object.notifications.filter(email).length)
  http: ...
  sms: ...
}
```